lua/harpoon/init.lua

```diff
@@ -66,11 +71,13 @@ local function ensure_correct_config(config)
 
     local proj = projects[vim.loop.cwd()]
     if proj.mark == nil then
-        proj.mark = {marks = {}}
+        log.trace("ensure_correct_config(): No marks found for", vim.loop.cwd())
+        proj.mark = { marks = {} }
```

Comment:
Formatting change.

Is this a formatter you are using or manual?  If there is a formatter you are using, what is the settings?  If there are settings, make the formatting changes + the formatter settings as a PR!!  That way we can just have uniform spaced code.